chain-extensions/pallet-assets/src/mock.rs

```diff
@@ -115,6 +115,7 @@ impl pallet_contracts::Config for Test {
     type MaxDelegateDependencies = MaxDelegateDependencies;
     type Migrations = ();
     type RuntimeHoldReason = RuntimeHoldReason;
+    type Xcm = ();
```

Comment:
Any potential future work to explore here?

Reply:
If we `activate` it then we will need to support it (docs, SDK, builders questions, maintenance). Let's see if demand is growing for it

Reply:
Of course, I don't believe we should pursue it atm.